companion/manifest.json

```diff
@@ -0,0 +1,26 @@
+{
+	"id": "monteiro-pptvideoosc",
+	"name": "monteiro-pptvideoosc",
+	"shortname": "monteiro-pptvideoosc",
```

Comment:
the shortname is the same as the name. it should be shorter, just the minimum to recognize the connection